topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -122,14 +122,15 @@ def populate_item(self, metadata_row: Dict[str, str], asset: Optional[Asset] = N
         item.add_extension(StacExtensions.version.value)
         item.add_extension(StacExtensions.processing.value)
 
-    def get_title(self, survey: str, alternate_survey_name: str) -> Union[str, None]:
-        if not survey or survey == "0" or survey == "":
-            if alternate_survey_name and alternate_survey_name != "":
-                return alternate_survey_name
-            else:
-                return None
-        else:
-            return survey
+    def get_title(self, survey: str) -> str:
+        survey_names = get_metadata(DataType.SURVEY_FOOTPRINT_HISTORIC)
+        title: str = ""
+
+        if len(survey_names) == 0:
+            raise Exception(f"No name found for survey {survey}")
```

Comment:
Because we're using csvs, is there the possibility that there is no survey name but this is represented with an empty str?
I think the empty str would still pass through here and we'd get a collection with no title.

Reply:
Good point @MDavidson17 - I've fixed that now.